tools/send-to-caas/send-utils.js

```diff
@@ -193,9 +193,8 @@ const getOrigin = (fgColor) => {
     cc: 'hawks',
     dc: 'doccloud',
   };
-  const originLC = (mappings[origin] || origin).toLowerCase();
-  if (originLC) {
-    return originLC;
+  if (mappings[origin.toLowerCase()] || origin) {
+    return mappings[origin.toLowerCase()] || origin;
```

Comment:
you might want to use a constant here
```suggestion
    const originLC = mappings[origin.toLowerCase()] || origin;
    if (originLC) return originLC;
```